source/features/minimize-upload-bar.tsx

```diff
@@ -0,0 +1,39 @@
+import './minimize-upload-bar.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import features from '../libs/features';
+import * as icons from '../libs/icons';
+
+async function addButton(): Promise<void> {
+	for (const toolbar of select.all('form:not(.rgh-has-upload-field) markdown-toolbar')) {
+		select(':scope > div:last-child', toolbar)!.prepend(
+			<button type="button" className="toolbar-item tooltipped tooltipped-n rgh-upload-btn" aria-label="Attach files">
+				{icons.cloudUpload()}
+			</button>
+		);
+		toolbar.closest('form')!.classList.add('rgh-has-upload-field');
+	}
+}
+
+function triggerUpload(event: KeyboardEvent): void {
+	(event.target as Element)
+		.closest('form')!
```

Comment:
```suggestion
		.form!
```

Reply:
I tried to the same here https://github.com/sindresorhus/refined-github/pull/2157/files#diff-94fcd77efaf038ad2377d40331fc7611R15 without success. Any suggestions?